client/plots/matrix.interactivity.js

```diff
@@ -274,6 +274,9 @@ export function setInteractivity(self) {
 		// clicking only show actions for available genomic data; can later expand to non-genomic data and custom overrides
 		const q = self.state.termdbConfig.queries
 		if (!q) return // no genomic queries
+
+		const showBrainImaging = JSON.parse(sessionStorage.getItem('optionalFeatures') || `{}`)?.showBrainImaging
```

Comment:
next can centralize this logic in routes/termdb.config.ts, so that client can simply check for keys in queries{}